runtime/moonbeam/tests/integration_test.rs

```diff
@@ -2448,23 +2448,20 @@ fn make_sure_glmr_cannot_be_transferred_precompile() {
 					id: [1u8; 32],
 				}),
 			};
-			assert_noop!(
-				XTokens::transfer_multiasset(
-					origin_of(AccountId::from(ALICE)),
-					Box::new(VersionedMultiAsset::V1(MultiAsset {
-						id: Concrete(moonbeam_runtime::xcm_config::SelfLocation::get()),
-						fun: Fungible(1000)
-					})),
-					Box::new(VersionedMultiLocation::V1(dest)),
-					40000
-				),
-				orml_xtokens::Error::<Runtime>::XcmExecutionFailed
-			);
+			assert_ok!(XTokens::transfer_multiasset(
+				origin_of(AccountId::from(ALICE)),
+				Box::new(VersionedMultiAsset::V1(MultiAsset {
+					id: Concrete(moonbeam_runtime::xcm_config::SelfReserve::get()),
```

Comment:
```suggestion
					id: Concrete(moonbeam_runtime::xcm_config::SelfLocation::get()),
```
This changed from SelfLocation to SelfReserve in the test. 

Not sure why but feel free to ignore if it doesn't matter, I'll look up the definitions.

Reply:
Yeah the previous version (with SelfLocaiton) was wrong. SelfLocation represents the parachain, SelfReserve the token of the parachain. Although in the end (since none of those two multilocations was enabled) the error was identical, thats why I did not detect it before

Reply:
Thanks for explaining!